<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>840</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>840</width>
    <height>520</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(249, 249, 249);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QVBoxLayout" name="verticalLayout" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>141</width>
      <height>131</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>141</width>
      <height>131</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonTangentialStress">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>221</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>221</width>
      <height>61</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 64);</string>
    </property>
    <property name="text">
     <string>Simulación de Esfuerzos Tangenciales</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRadialStress">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>400</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>191</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>191</width>
      <height>61</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 64);</string>
    </property>
    <property name="text">
     <string>Simulación de Esfuerzos Radiales</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDifussion">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>400</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>151</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>151</width>
      <height>61</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 64);</string>
    </property>
    <property name="text">
     <string>Simulación de Difusión</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonComparison">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>400</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>151</width>
      <height>61</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>151</width>
      <height>61</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 64);</string>
    </property>
    <property name="text">
     <string>Comparar Resultados</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>210</y>
      <width>231</width>
      <height>151</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>231</width>
      <height>151</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>231</width>
      <height>151</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 122);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Juan David Parra&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Valentina Galindo&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Melissa Osorio &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Dayana Gomez &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>601</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>601</width>
      <height>81</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>601</width>
      <height>81</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);

color: rgb(0, 0, 122);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt; font-weight:600;&quot;&gt;Simulación del transporte de hidrogeno &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVBoxLayout</class>
   <extends>QWidget</extends>
   <header>qvboxlayout.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
